corehq/apps/hqwebapp/views.py

```diff
@@ -533,12 +534,20 @@ def logout(req, default_domain_redirect='domain_login'):
 @location_safe
 @two_factor_exempt
 def ping_response(request):
+    app_id = request.GET.get('selected_app_id', '')
+    new_app_version_available = False
+    if app_id:
```

Comment:
It'd be nice to avoid pulling the whole app doc. Could you use `get_latest_released_build_id` or other dbaccessors?

Reply:
Ohh! Thanks. That would be nice. I was looking for something like that.

Reply:
I think other dbaccessors require a domain in order to get the id but we don't have a domain info available in the request. 
We can definitely pass in the domain from the front-end and then use the dbaccessors. 
Do you think that it would be a better approach?

Reply:
ah, I missed that domain wasn't here, but yeah, it seems doable to pull it out of the URL. You'd need to regex the URL instead of parsing it out of the hash fragment, right? Or pass it as initial page data? That seems a little messier than pulling the app id, but I do think it'd be good to not deal with the entire app doc.

It'd be nice for the js to have access to web apps js, since that has the app and domain easily available - like, imagine a web apps inactivity js module that extends the inactivity.js module. But I haven't thought through that at all, feel free to ignore.

Reply:
The domain is available in initial page data so it would be an easier way to go. I tried to make it work with the `get_latest_released_build_id `, I realized what we have here is the app build id and not the actual `app_id`. I looked for dbaccessors which provides us with app_id if build_id is passed but did not find any. 
I did find `get_app_cached` which I have used to find the actual id in https://github.com/dimagi/commcare-hq/pull/30778/commits/90235e5e948ea5564a0790d46ad408bf4102b059

Reply:
Updates look good 👍